Package.swift

```diff
@@ -23,6 +23,7 @@ let package = Package(
             url: "https://github.com/apple/swift-argument-parser", .upToNextMajor(from: "1.5.0")
         ),
         .package(url: "https://github.com/apple/swift-log", .upToNextMajor(from: "1.6.3")),
+        .package(url: "https://github.com/tuist/path", .upToNextMajor(from: "0.3.8")),
```

Comment:
nit: should we use upToNextMinor for `0.x` versions?

Reply:
Good idea. I'll update it.